Dockerfile

```diff
@@ -41,9 +41,6 @@ ENV PATH=/opt/ncbi-blast-2.11.0+/bin:$PATH
 WORKDIR /opt
 RUN git clone https://github.com/marcelauliano/MitoHiFi.git && \
     cd MitoHiFi && \
```

Comment:
When installing from GitHub (but not from a specific version) it might be a good idea to pin to a specific commit. It looks like this: `git reset --hard 1926400fcb098ab3e4745ea0872d23f91d730340`

Check here (https://github.com/human-pangenomics/hpp_production_workflows/blob/master/QC/docker/merqury/Dockerfile) for an example.

In order to find the commit, just go to the GitHub repo, and click on the commits area. Then you can find a suitable commit (the most recent one, in this case).

This helps with reproducibility, and I guess to some extent protects you from changes in their code in the off-chance that you need to rebuild your Docker image.

Reply:
Added this for both MitoHiFi and Mitofinder now